markov.js

```diff
@@ -1,4 +1,36 @@
-function buildChains(corpus, chainLength) {
+const fixCapitalisation = sentences => {
+    for (let i = 0; i < sentences.length; i++) {
+        sentences[i][0] = sentences[i][0].charAt(0).toUpperCase() + sentences[i][0].slice(1);
+        for (let j = 0; j < sentences[i].length; j++) {
```

Comment:
Its a search for "i" and "i'll" which was extended to all the others. I can remove them if you want?